about.html

```diff
@@ -58,16 +86,22 @@
 
       <div class="btn-box">
         <button class="switch-mode">
-          <img src="./img/btn-icons/switch.png" alt="" />
+          <svg class="icon switch-ico">
```

Comment:
The same as above - no need to reduce class name by just one letter. It makes sense for example in case `button` -> `btn`

Reply:
#fixed